data/World/Ganons Castle.json

```diff
@@ -48,6 +50,11 @@
             "Fire Trial Clear": "
                 can_use(Goron_Tunic) and can_use(Golden_Gauntlets) and 
                 can_use(Light_Arrows) and can_use(Longshot)"
+        },
+        "locations": {
+            "Ganons Castle Fire Trial Recovery Heart": "logic_fewer_tunic_requirements or can_use(Goron_Tunic)",
```

Comment:
This recovery heart might be safe even without FTR. We only promise a tunic for heated rooms of the Fire Temple, but maybe the Fire Trial should count also? You do generally have a tunic at this point in normal play, and it's an area with Fire in the name.

If you decide fire trial should also promise you a tunic with FTR off, you might need to modify the tags/tooltip(?) of FTR to include Ganon's Castle. (Note that I recently modified it to tag Crater also. It sort of sometimes applies there too, so I thought it should have that tag.)